content/tutorials/quickstart/mint-and-burn-nftokens.md

```diff
@@ -219,51 +220,51 @@ Disconnect from the ledger.
 // ******************* Get Tokens ************************
 // *******************************************************
 
-      async function getTokens() {
+async function getTokens() {
 ```
 
 
 Connect to the ledger and get the account wallet.
 
 
 ```
```

Comment:
Another suggestion not important immediately: these code samples are a good use case for the `include_code` macro. That way you don't have to update the code in two places when changes are needed.

Also, syntax highlighting will work more reliably if you specify the language in the opening fence, e.g. \`\`\`js instead of just ```